spec/features/user_visits_brewpub_index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature "user visits brewpub index" do
+
+  scenario "user visits the index page" do
+    brewpub = create(:brewpub)
+    visit "/"
+
+    expect(page).to_not have_content("Yards")
+    expect(page).to_not have_content("901 N Delaware Ave")
+    expect(page).to_not have_content("Philadelphia")
+    expect(page).to_not have_content("PA")
+    expect(page).to_not have_content("19123")
+    expect(page).to_not have_content("A beer co.")
```

Comment:
If this is the sad path for visiting the root, make sure you have some tests for the happy path.  As in, what SHOULD be on the page for the root path.